src/components/Sherlock/Sherlock.js

```diff
@@ -1,5 +1,4 @@
 import React, { Component } from 'react'
-import PropTypes from 'prop-types';
```

Comment:
I was leaving this here so that we create the proptypes for this component. So instead of removing it, let's implement the proptypes.